templates/search/results-en.html

```diff
@@ -17,7 +17,7 @@
 <form>
 	<label class="wb-inv" for="sch-inp">Search Government of Canada websites</label>
 	<div class="input-group mrgn-tp-lg">
-		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" />
+		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query  name="q" />
```

Comment:
Suuuuper small request. Can you please remove one space just for the purpose of having consistent clean code? Not an error, I<m just really picky sometimes 😛 

```suggestion
		<input spellcheck="false" autocomplete="off" id="sch-inp" class="form-control" type="search" data-fusion-query name="q" />
```